environ.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"os"
+	"strings"
+)
+
+const clientsKey = "XLANG_CLIENTS"
+const paramKey = "XLANG_PARAMS_"
```

Comment:
Nit: more descriptive names for globals to avoid namespace collisions. Like, `paramKeyPrefix`. Also consider perpending `_` to the const name if it's used only in this file.